Sources/FluidAudio/Diarizer/DiarizerTypes.swift

```diff
@@ -1,12 +1,20 @@
 import Foundation
 
+public enum EmbeddingMode: String, Sendable, CaseIterable {
+    case full = "full"  // Full embedding extraction (10.5s)
+    case fast = "fast"  // Fast mode with optimizations
+    case skip = "skip"  // Skip embedding extraction entirely
+    case cached = "cached"  // Use cached embeddings if available
+}
```

Comment:
Do we need these types in this PR? This is probably better suited for the streaming change?